src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,68 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class WorkWithFile {
+    private static final String SUPPLY_WORD = "supply,";
+    private static final String BUY_WORD = "buy,";
+    private static final String RESULT_WORD = "result,";
```

Comment:
Would be better to create a COMMA_SEPARATOR constant and rename those ones to SUPPLY_OPERATION